pegasus/sites.v3/code.org/public/hourofcode/flappy.md.partial

```diff
@@ -2,11 +2,14 @@
 title: Teacher Resources - Make a Flappy Game
 nav: hoc_teacher_resources_nav
 theme: responsive
+video_player: true
 ---
 
 # Make a Flappy Game: An Hour of Code Tutorial
 
-<center><iframe style="max-width:100%" width="600" height="337" src="https://www.youtube.com/embed/VQ4lo6Huylc" frameborder="0" allowfullscreen></iframe></center>
+<div style="width:600px; max-width:100%; margin-left:auto; margin-right:auto;">
```

Comment:
any reason not to put this HTML in the view as well?

Reply:
I thought a video partial view can be used in other contexts, so I separate it from its container div.

Reply:
I can see that, but I think in this context it makes sense to keep them together.

The specific motivation here is to keep complicated stuff like html out of the hands of translators, to minimize confusion (they tend to translate things like HTML tag names). In the case where we _do_ want to reuse this in another context, it wouldn't be too much work to separate the template out at that point rather than doing it now

Reply:
I can do that to make the markdown files cleaner. 

For my own education, does CrowdIn automatically display (and hide) simple HTML elements from translator? For example, this file is full of HTML tags https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/public/athome.md.partial. But in CrowdIn, the file is displayed quite nicely without exposing those tags to translator https://crowdin.com/translate/codeorg-markdown/177852/en-vi?filter=basic&value=0.

Reply:
Crowdin _sometimes_ automatically hides HTML elements in that view, but doesn't always. If you look through the individual strings in that file, you will find several that turn the embedded tags into `<0>` and `</0>`, but it doesn't do so consistently even on the same string.

Note also that this display smoothing is a property of the Crowdin editing UI, and so in addition to being inconsistent it doesn't apply to our translators who work outside of Crowdin, like Translate By Humans.

Reply:
Got it! Thanks Elijah. I've pushed a new commit moving the HTML tags to ERB files; awaiting for the test run. Anything else you think I should change?